src/openrct2/platform/Crash.cpp

```diff
@@ -318,9 +320,10 @@ static bool OnCrash(
 
 static std::wstring GetDumpDirectory()
 {
-    char userDirectory[MAX_PATH];
-    platform_get_user_directory(userDirectory, nullptr, sizeof(userDirectory));
-    auto result = String::ToWideChar(userDirectory);
+    auto env = GetContext()->GetPlatformEnvironment();
```

Comment:
I think this should read

```suggestion
    auto env = OpenRCT2::GetContext()->GetPlatformEnvironment();
```

Reply:
Oddly everything except MSVC built this, on my end it also did but yeah.